omim2obo/main.py

```diff
@@ -243,6 +241,10 @@ def run(use_cache: bool = False):
             chr_id = '9606chr' + cyto_location
             # What's RO['0002525'] - joeflack4 2021/11/11
             graph.add((OMIM[mim_number], RO['0002525'], CHR[chr_id]))
+        if mim_data['phenotype_mapping_info_label']:
+            # todo: store mim_data['phenotype_mapping_info_key'] in provenance somehow?
+            # https://biolink.github.io/biolink-model/docs/GeneToDiseaseAssociation.html
+            graph.add((OMIM[mim_number], BIOLINK['GeneToDiseaseAssociation'], BIOLINK['has_evidence']))
```

Comment:
@cmungall @matentzn This is the new information that I'm adding which I was able to extract from `morbidmap.txt`, which probably fully addresses #73. I looked at the code that Kevin Schaper shared with me. His code used Koza to do add this information, but I believe the result is the same.

Example from new `omim.ttl`:
```ttl
OMIM:109480 a owl:Class ;
    rdfs:label "BSG" ;
   ...
    oboInOwl:hasExactSynonym "BSG",
        "basigin",
        "cd147 antigen",
        "extracellular matrix metalloproteinase inducer",
        "m6 leukocyte activation antigen",
        "tcsf" ;
   ...
    biolink:GeneToDiseaseAssociation biolink:has_evidence ;
    biolink:category biolink:Gene .
```

Reply:
Discussed this with Nico, Kevin Schaper, and Sabrina today. Looks like `koza` is fetching mostly the same type of info here, except that there has been some configuration to load up some manually curated ECO mappings on each of the 4 OMIM phenotype mappings. Nico came to determination that I can add the literal text of the phenotype mapping descriptions, and that will be fine for now.